pkgs/tools/security/bitwarden_rs/vault.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
```

Comment:
Since a recent change in nixpkgs, `name` automatically gets set to this if you have `pname` and `version` defined, so this can be removed. Also for the other package.